scopes/react/react/templates/react-env/files/main.runtime.ts

```diff
@@ -5,6 +5,7 @@ export function mainRuntimeFile({ namePascalCase: Name, name }: ComponentContext
 import { ReactAspect, ReactMain } from '@teambit/react';
 import { EnvsAspect, EnvsMain } from '@teambit/envs';
 import { ${Name}Aspect } from './${name}.aspect';
+import { previewConfigTransformer, devServerConfigTransformer } from './webpack/webpack-transformers';
```

Comment:
I think you can delete line 14 in connection with this change - it looks like leftovers.

Reply:
Thanks, I comment out this line, because this is the import of the Webpack transformer that can be used in line 31.